### PR TITLE
FLJ-10 Added post provisionning setup for node/gulp/jspm

### PR DESCRIPTION
**Pull request by Phi-D**

### CODE REVIEW

**Comment by PierrePaul** [comment]
Sweet. Thank you.

File: provisioning/bootstrap.yml [hunk 1]
```diff
@@ -10,3 +10,49 @@
...
+        shell:
+          executable=/bin/bash
+          npm install -g gulp
+
+      - name: Install jspm globally
+        shell:
+          executable=/bin/bash
+          npm install jspm -g
+
+      - name: Lock jspm
```
Comment (pbuyle) [inline provisioning/bootstrap.yml:36]: Why is this needed in provisioning?
Running the command install jspm and put it in the `package.json` file (which is under version control). This is redundant with the `npm install` done above.
